base/bucket_gocb.go

```diff
@@ -169,14 +169,14 @@ func GetCouchbaseBucketGoCBFromAuthenticatedCluster(cluster *gocb.Cluster, spec
 	// to avoid gocb queue overflow issues
 	singleOpsQueue := make(chan struct{}, MaxConcurrentSingleOps*nodeCount*numPools)
 	bucketOpsQueue := make(chan struct{}, MaxConcurrentBulkOps*nodeCount*numPools)
-	viewOpsQueue := make(chan struct{}, MaxConcurrentViewOps*nodeCount)
+	viewOpsQueue := make(chan struct{}, spec.ViewQueryMaxConcurrentOpsPerNode*nodeCount)
```

Comment:
I don't think we should multiple by node count here, as the limit we're trying to avoid is the number of sockets being used on this node, not the CBS node.  (this differs from kv ops, where we're limiting the number of concurrent ops in the per-node pipeline established to CBS nodes).